webapp/channels/src/components/post_view/embedded_bindings/select_binding/select_binding.tsx

```diff
@@ -171,7 +171,7 @@ class SelectBinding extends React.PureComponent<Props, State> {
                         onSelected={this.handleSelected}
                         placeholder={label}
                         inputClassName='post-attachment-dropdown'
-                        value={this.state.selected?.text}
+                        value={this.state.selected?.text ?? ''}
```

Comment:
If we make `value` optional, this change will not be needed.

Reply:
Value is now optional, change reverted.